### PR TITLE
Sequence viewer options block select fix

### PR DESCRIPTION
**Pull request by duncanmeech**
#### Overview

https://zube.io/autodesk/genome-designer/c/1137

Let extensions know which was the last selection option versus just sending all the currently selected options/blocks ids

#### Requirements

- [ ] Adds a test (for features + fixes)
- [ ] Docs updated (for features + fixes)

### CODE REVIEW

File: src/actions/focus.js [hunk 1]
```diff
@@ -231,6 +231,10 @@ export const focusBlockOption = (blockId, optionId) => {
     const oldOptions = getState().focus.options;
     const options = Object.assign({}, oldOptions, { [blockId]: optionId });
 
+    // options is all the focused blocks and associated selected option. For extensions etc
+    // we need to show which block was most recently selected.
+    options.selectedOptionId = optionId;
```
Comment (maxbates) [inline src/actions/focus.js]: This is already available at focus.optionId - you can't add this to the map as it is used many places and expects only blockIds

**Comment by maxbates** [comment]
not as is

**Comment by maxbates** [comment]
I'm watching your `!important` in CSS

File: src/actions/focus.js (hunk 1 above)
Comment (duncanmeech) [inline src/actions/focus.js]: If it exists it is not being passed to the extensions in the focus object. There is focus.options but it does not give any indication of the most recently selected.

**Pull request closed without merging**